### PR TITLE
Update pytests

### PR DESCRIPTION
**Pull request by zhx828**

### CODE REVIEW

File: test_Annotation.py [hunk 1]
```diff
@@ -16,21 +16,18 @@
 from AnnotatorCore import HGVSgQuery
 from AnnotatorCore import ReferenceGenome
 
-ONCOKB_API_TOKEN = os.environ["ONCOKB_API_TOKEN"]
+ONCOKB_API_TOKEN = '5d682637-c3c8-4353-b623-347d8f8ca5c2'
```
Comment (calvinlu3) [inline test_Annotation.py:19]: I think you accidently committed your token.

File: test_Annotation.py [hunk 2]
```diff
@@ -16,21 +16,18 @@
...
-HIGHEST_DX_LEVEL_INDEX = HIGHEST_LEVEL_INDEX + 7
-HIGHEST_PX_LEVEL_INDEX = HIGHEST_DX_LEVEL_INDEX + 5
+MUTATION_EFFECT_INDEX = VARIANT_EXISTS_INDEX+1
+ONCOGENIC_INDEX = MUTATION_EFFECT_INDEX+2
+LEVEL_1_INDEX = ONCOGENIC_INDEX+1
+LEVEL_2_INDEX = LEVEL_1_INDEX+1
+LEVEL_3A_INDEX = LEVEL_1_INDEX+2
+HIGHEST_LEVEL_INDEX = LEVEL_1_INDEX+7
+HIGHEST_DX_LEVEL_INDEX = HIGHEST_LEVEL_INDEX+7
+HIGHEST_PX_LEVEL_INDEX = HIGHEST_DX_LEVEL_INDEX+5
```
Comment (calvinlu3) [inline test_Annotation.py:30]: flake8 is complaining about no spaces around `+` sign.

File: test_Annotation.py (hunk 1 above)
Comment (calvinlu3) [reply test_Annotation.py:19]: Also think the tests are failing because token was removed here.

**Review by calvinlu3 — CHANGES_REQUESTED** [review]

File: test_Annotation.py (hunk 1 above)
Comment (zhx828) [reply test_Annotation.py:19]: 😞 I will fix

Comment (zhx828) [reply test_Annotation.py]: @calvinlu3 the reason the tests failing is because the fork branch cannot use the main repo secret.

**Review by calvinlu3 — APPROVED** [review]
Looks good

**Pull request merged**